ajax/libs/Embetty/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Embetty",
+  "description": "Embetty displays remote content like tweets or videos without compromising your privacy.",
+  "version": "1.2.0",
```

Comment:
```suggestion
```

Remove `version` attr, the auto-update bot will populate this once it pulls down versioned library files.